### ISSUE TITLE
Failing to render GraphQL Query requests

### ISSUE THREAD

**Issue by camiloclc** [getinsomnia/insomnia#6858]
### Expected Behavior

Open a request and being able to use it

### Actual Behavior

When opening a GraphQL request, I get an alert modal of an Application Error, described as "Failed to render Hl".

The Stack Trace is as follows:

```
TypeError: e.includes is not a function
    at KVe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:470:868)
    at _l (file:///Applications/Insomnia.app/Contents/Resources/app.asar/debug-672b1df6.js:53:34237)
    at FS (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:42:19553)
    at aF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:44101)
    at tF (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39819)
    at qpe (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39745)
    at Fd (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:39596)
    at Px (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:35959)
    at QO (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:44:34906)
    at N (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-37d8fc4b.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)
```

The collection requests list and the response pane render successfully, but the middle pane where the request context is defined just says `Render Failure: e.includes is not a function`

### Reproduction Steps

1 - Open collection
2 - Open a request identified as GraphQL Query

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

8.4.4

### What operating system are you using?

macOS

### Operating System Version

macOS Sonoma 14.1.1

### Installation method

brew

### Last Known Working Insomnia version

8.4.2

**Comment by camiloclc** [comment]
Seems related to #6857. Guess @prerakathighnoteplatform was creating that issue while I was beggining to create this one.

**Comment by WallacyFrancis** [comment]
I'm having the same problem using an Ubuntu Linux OS! It occurred after downloading the latest update.

```
Version: Insomnia 8.4.4
Build date: 22/11/2023
OS: Linux x64 5.15.0-89-generic
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: x64
```

In a failed attempt, I tried to reinstall Insomnia but the error remains

**Comment by sarahzhao25** [comment]
I'm also having the same problem after downloading the latest update!

```
Version: Insomnia 8.4.4
Build date: 22/11/2023
OS: macOS Ventura 13.6.1
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64

**Comment by xplo8** [comment]
Also having this issue after last update 
```
Version: Insomnia 8.4.4
Build date: 11/22/2023
OS: Darwin arm64 22.6.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64
```

**Comment by marckong** [comment]
Anyone who would be willing to jump on a call and help us troubleshoot? Couldn't reproduce it locally on my machine.

**Comment by camiloclc** [comment]
I could do it in a couple of hours

**Comment by marckong** [comment]
@camiloclc are you on our community slack group? Thank you for the help!

**Comment by mealbarracin10** [comment]
I'm having the same issue, any ideas any ideas how to solve that please

**Comment by mealbarracin10** [comment]
Version: Insomnia 8.4.3
Build date: 11/22/2023
OS: Darwin x64 22.6.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: x64

**Comment by jlordupgrade** [comment]
ditto. Can't view any of my queries right now.

Version: Insomnia 8.4.4
Build date: 11/22/2023
OS: Darwin arm64 22.6.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64

**Comment by paulmerlyn** [comment]
Same error for me. Happy to jump on a call @marckong :

Version: Insomnia 8.4.4
Build date: 11/22/2023
OS: Darwin arm64 23.1.0
Electron: 27.0.3
Node: 18.17.1
Node ABI: 118
V8: 11.8.172.16-electron.0
Architecture: arm64

<img width="1086" alt="Screenshot 2023-11-22 at 1 53 46 PM" src="https://github.com/Kong/insomnia/assets/2975054/a883a970-27c5-4c80-bcf8-34e5bb528d3c">

**Comment by grosualexandru89** [comment]
I had the same problem after upgrading to 8.4.4, I tried to revert to 8.4.3 and still the same. Only after reverting to 8.4.2 it works as before the upgrade. 
https://github.com/Kong/insomnia/releases/tag/core%408.4.2

**Comment by AlexWilson-GIS** [comment]
I'll throw the Windows error + stack trace onto the pile for good measure:
![image](https://github.com/Kong/insomnia/assets/74787446/800a5f75-6cb0-40a7-b648-0a8823759788)
```
TypeError: e.includes is not a function
    at qVe (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:470:868)
    at _l (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/debug-1d1cf6bf.js:53:34237)
    at FS (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:42:19553)
    at oF (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:44101)
    at rF (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39819)
    at Gpe (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39745)
    at Bd (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:39596)
    at Px (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:35959)
    at eF (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:44:34906)
    at N (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:29:1615)
    at Immediate.me [as _onImmediate] (file:///C:/Users/ale11151/AppData/Local/insomnia/app-8.4.4/resources/app.asar/index-12757498.js:29:1987)
    at process.processImmediate (node:internal/timers:476:21)
```

**Comment by marckong** [comment]
Hi all, thanks for reporting these issues. Can you guys ping me on our community slack, please?

Mark Kim - and we can share the zoom link.

Thanks for all trying to help us troubleshoot this!

**Comment by camiloclc** [comment]
> @camiloclc are you on our community slack group? Thank you for the help!

I've just joined

**Comment by marckong** [comment]
It seems that the issue may occur from either of these two places. I would appreciate help to reproduce it.

https://github.com/Kong/insomnia/blob/da01223886a03583100a86aa3c97311ad42a64e6/packages/insomnia/src/ui/components/panes/request-pane.tsx#L67

https://github.com/Kong/insomnia/blob/develop/packages/insomnia/src/ui/components/editors/body/body-editor.tsx#L37

**Comment by marckong** [comment]
@camiloclc Thank you so much for the help on reproducing this issue!!!

**Comment by marckong** [comment]
For visibility @subnetmarco, this is the root cause of this issue - https://github.com/Kong/insomnia/blob/da01223886a03583100a86aa3c97311ad42a64e6/packages/insomnia/src/ui/components/editors/body/graph-ql-editor.tsx#L629

The value is passed as JSON when it requires string and refers to String.includes() method. Obviously, Object does not have .includes and throws error.

**Comment by marckong** [comment]
I put a fix for this in PR - https://github.com/Kong/insomnia/pull/6861#issue-2007272238. There are some TODOs left for unit tests. Depending on the product pressure, we will add more tests around this.

**Comment by filfreire** [comment]
Hey folks, we will launch a new release with a fix for this today, 8.4.5.

Closing this as a duplicate of #6857
